docs/ibc/integration.md

```diff
@@ -37,6 +46,9 @@ var (
     capability.AppModuleBasic{},
     ibc.AppModuleBasic{},
     transfer.AppModuleBasic{}, // i.e ibc-transfer module
+
+    // register light clients on IBC
+    ibctm.AppModuleBasic{},
```

Comment:
It might be helpful to show the import statement as well

Reply:
good call!